docs/federated_a_crd.md

```diff
@@ -0,0 +1,358 @@
+# Federate a CRD in the target cluster
+
+If you want to federate a CRD into your target cluster then the following example will help you achieve the same.
+
+## Getting Started
+
+These instructions will provide the process of federating a CRD across the target clusters.
+
+### Prerequisites
+
+You will need to set up federation v2 on one of the clusters. Please follow the instructions mentioned in our [user guide](https://github.com/kubernetes-sigs/federation-v2/blob/master/docs/userguide.md) for deploying federation v2. 
```

Comment:
As federation now support installation for both cluster-scope and namespace-scope, you are using cluster-scope in this example, so how about highlight that in this guide, you are using cluster-scoped federation and deploying federation controller in namespace `federation-system`? This can echo line 289 in this document.